<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceLinkClass</class>
 <widget class="QDialog" name="ReferenceLinkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>132</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>136</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReferenceLink</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblURL">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txtURL"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnTest">
        <property name="text">
         <string>Test Link</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="txtDescription"/>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>132</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ReferenceLinkClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
